tests/tests_shared.c

```diff
@@ -33,39 +33,39 @@
 #include "tests_shared.h"
 
 void print_intro(size_t tgt_tests) {
-    (void)printf(WHITEB("\n" ULINE("libsir") " %s (%s) running %zu %s...") "\n",
+    (void)printf(SIR_WHITEB(SIR_EOL SIR_ULINE("libsir") " %s (%s) running %zu %s...") SIR_EOL,
         sir_getversionstring(), (sir_isprerelease() ? "prerelease" : "release"),
         tgt_tests, TEST_S(tgt_tests));
 }
 
 void print_test_intro(size_t num, size_t tgt_tests, const char* const name) {
-    (void)printf(WHITEB("\n(%zu/%zu) '%s'...") "\n\n", num, tgt_tests, name);
+    (void)printf(SIR_EOL SIR_WHITEB("(%zu/%zu) '%s'...") SIR_EOL SIR_EOL, num, tgt_tests, name);
 }
 
```

Comment:
These 3 are changing the order of the newline/color start token. Should either not do that or run it and make sure it does not change the look of the output

Reply:
This is a case where I think the macro did the right thing.  We shouldn't be coloring invisible newlines.  The color should start and end with the text to be colorized.  I did run it and it does look the same.